lib/generated/l10n.dart

```diff
@@ -14,22 +14,23 @@ import 'intl/messages_all.dart';
 
 class S {
   S();
-  
+
```

Comment:
I think you auto-formatted this file. Please don't, or we'll get new diffs whenever it's generated again.